platforms/web/lib/useListeners/utils.ts

```diff
@@ -17,6 +17,10 @@ limitations under the License.
 import { ACTION_TYPES } from '../constants';
 import { ActionTypes, ActionState, AllActionStates } from '../types';
 
+/**
+ * Create the default state for all the available actions
+ * @returns {AllActionStates}
```

Comment:
I saw comments in the react-sdk, I though it will be a good idea but it's kind of duplicate with TS definition

Reply:
Unless there is a really good reason, I think we should leave them out because they will rot.

I expect the ones in react-sdk are left over from when it was JavaScript...

Reply:
Make sense, will remove them!